robocopkaka/bootcamp-29-project
# #158176735 See user events

#### What does this PR do?
show user events on their profile
#### Description of Task to be completed?
* create endpoint for retrieving events for a user
* write action creator that utilizes the endpoint
* modify `UserProfile` component to make use of the action creator
* write tests for the `UserProfile` component
* allow users delete their own events
* write e2e test for UserProfile
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to localhost:8000 on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* click on `Login`
* enter your details
* click on `Login`
* click on the `Profile` dropdown
* click on `Profile`
* you should see events that you've created
#### Any background context you want to provide?
e2e test doesn't work properly for UserProfile
#### What are the relevant pivotal tracker stories?(if applicable)
#158176735
